pgeocode.py

```diff
@@ -233,7 +233,10 @@ def _get_data(country: str) -> Tuple[str, pd.DataFrame]:
                     dtype={"postal_code": str},
                 )
             if not os.path.exists(STORAGE_DIR):
-                os.mkdir(STORAGE_DIR)
+                try:
+                    os.mkdir(STORAGE_DIR)
+                except FileExistsError:
+                    pass
```

Comment:
Thanks! This would a bit simpler. Otherwise LGTM.

```suggestion
            os.makedirs(STORAGE_DIR, exists_ok=True):
```